python/bolt-mcp/tests/integration/test_mcp_sse_integration.py

```
@@ -0,0 +1,94 @@
+mount_mcp(api, mcp)
+"""
+
+
+def _json_headers(session_id=None):
+    h = {"Accept": DUAL_ACCEPT, "Content-Type": "application/json", "MCP-Protocol-Version": "2025-06-18"}
+    if session_id:
+        h["Mcp-Session-Id"] = session_id
+    return h
+
+
+def _encode(method, params=None, *, id=1, session_id=None):
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `_encode`

(ANN202)